inc/simple_UART.h

```diff
@@ -8,14 +8,13 @@
 #ifndef SIMPLE_UART_H_
 #define SIMPLE_UART_H_
 
-#include "stm32f4xx.h"
-#include "Buffer.h"
 
-#define MAX_OUPUT_DATA (10)
+#include "Buffer.h"
 
 Buffer inputBuffer;
```

Comment:
I know that this isn't really part of your commit, but I noticed while it's here. Global variables should be allocated in ".c" files, and if you want to share them, you should use the keyword extern before the variable type, otherwise you'll be redefining the variable. This could have been there for a while, but do you think you'd be able to change it? It should only be declared once in a .c (without extern) and can then be shared in headers etc using the extern keyword